requirements/base.txt

```diff
@@ -4,49 +4,37 @@
 #
 #    pip-compile --allow-unsafe --output-file=requirements/base.txt requirements/base.in
 #
-appdirs==1.4.4            # via virtualenv
 attrs==20.3.0             # via pytest
 awesome-slugify==1.6.5    # via -r requirements/base.in
 bcrypt==3.2.0             # via paramiko
 behave==1.2.6             # via -r requirements/base.in
-certifi==2020.12.5        # via requests
 cffi==1.14.4              # via bcrypt, cryptography, pynacl
-chardet==3.0.4            # via requests
 click-default-group==1.2.2  # via -r requirements/base.in
-click==7.1.2              # via -r requirements/base.in, click-default-group, pip-tools, rq
+click==7.1.2              # via -r requirements/base.in, click-default-group, pip-tools
 colorlog==4.7.2           # via -r requirements/base.in
 cryptography==3.4.7       # via paramiko
-distlib==0.3.1            # via virtualenv
 fabric3==1.14.post1       # via -r requirements/base.in
-filelock==3.0.12          # via tox, virtualenv
 graphql-core==3.1.2       # via sgqlc
-idna==2.10                # via requests
 iniconfig==1.1.1          # via pytest
-packaging==20.7           # via pytest, tox
+packaging==20.7           # via pytest
 paho-mqtt==1.5.1          # via -r requirements/base.in
 parameterized==0.7.4      # via -r requirements/base.in
 paramiko==2.7.2           # via fabric3
 parse-type==0.5.2         # via behave
 parse==1.18.0             # via behave, parse-type
 pip-tools==5.4.0          # via -r requirements/base.in
-pluggy==0.13.1            # via pytest, tox
-py==1.9.0                 # via pytest, tox
+pluggy==0.13.1            # via pytest
+py==1.9.0                 # via pytest
 pycparser==2.20           # via cffi
 pynacl==1.4.0             # via paramiko
 pyparsing==2.4.7          # via packaging
 pytest==6.1.2             # via -r requirements/base.in
-redis==3.5.3              # via -r requirements/base.in, rq
 regex==2020.11.13         # via awesome-slugify
-requests==2.25.1          # via -r requirements/base.in
```

Comment:
I see, so d3a-interface is installed via pip as a separate step and is not included in the requirements.in file. In this case all is well. We might think about harmonizing the way we import d3a-interface at some point (by adding it together with the other requirements in the .in file) however should not be done in this story since it is blocking people. I'll notify the next PR that meddles with requirements.